src/components/HomePage.js

```diff
@@ -0,0 +1,36 @@
+import React, { PropTypes, Component } from 'react';
+import { Link } from 'react-router';
+import '../stylesheets/Login.scss';
+import MainMenu from './MainMenu'
+class HomePage extends Component {
+
+  constructor(props) {
+    super(props);
+    this.openModal = this.openModal.bind(this);
```

Comment:
We have Babel ES6 preset, so you should be able to rewrite as `openModal = () => {}`, which would make this binding statement unncessary